sdk/daml-script/test/daml/upgrades/UnvettedPackages.daml

```diff
@@ -15,10 +15,7 @@ import DA.Time (seconds)
 
 -- These tests rely on unvetting, none of them will work on IDE
 main : TestTree
-main = tests $ []
-
--- TODO(https://github.com/DACH-NY/canton/issues/25366): restore after flakyness is fixed
-toRestore =
+main = tests
```

Comment:
I notice that the original PR was `main = tests $` - is that trailing `$` significant?